### PR TITLE
Fix LHSExperiment(size) constructor

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: TODO [hunk 1]
```diff
@@ -1 +1,2 @@
 Remove deprecated LAR class in 1.10 version
+Remove deprecated WeightedExperiment::getWeight
```
Comment (sofianehaddad) [inline TODO]: Where did you remove it?

**Comment by jschueller** [comment]
in #310 but I forgot to mention it

File: TODO (hunk 1 above)
Comment (jschueller) [inline TODO]: in #310 but I forgot to mention it

**Comment by sofianehaddad** [comment]
@jschueller no in #310 you deprecated it but it is not removed

**Comment by jschueller** [comment]
yes, deprecated for 1.8 and to be removed in the next version

**Comment by sofianehaddad** [comment]
ah sorry it is the TODO file

**Comment by sofianehaddad** [comment]
+1

**Comment by regislebrun** [comment]
+1

**Pull request merged**